misc/qutebrowser.spec

```diff
@@ -59,7 +59,8 @@ exe = EXE(pyz,
           debug=False,
           strip=False,
           upx=False,
-          console=False )
+          console=False,
+          version='misc/file_version_info.txt')
```

Comment:
I'd expect to see this documented [somewhere here](https://pyinstaller.readthedocs.io/en/latest/spec-files.html), but I don't - any idea where I can find documentation about it?

Reply:
The documentation is [here](https://pyinstaller.readthedocs.io/en/latest/usage.html?highlight=vsversion#capturing-windows-version-data) but the usage on spec files is missing. Command line usage is [here](https://pyinstaller.readthedocs.io/en/latest/usage.html?highlight=vsversion#windows-specific-options)